gax/operations_stub.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAPIC_GENERATOR_CPP_GAX_OPERATIONS_STUB_H_
+#define GAPIC_GENERATOR_CPP_GAX_OPERATIONS_STUB_H_
+
+#include "google/longrunning/operations.pb.h"
+#include "gax/call_context.h"
+#include "gax/status.h"
+
+namespace google {
+namespace gax {
+
+class OperationsStub {
```

Comment:
This is abigger topic (somewhat mentioned in the design doc), but how this would coexist with a GAPIC client stub, which will be generated by gapic-generator-cpp for the [Operations service in operations.proto](https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L54).

It must be possible to use the automatically generated stub instead of manually writing it (it is ok to have it manual for now, to unblock things, though).

Reply:
This is an interesting question. Does it need to coexist? We may choose never to generate a client and stub from operations.proto, and instead use the handwritten classes as our only solution. I'm not committed to that, but it is a possibility.

Reply:
+1 for just the handwritten. It makes testing easier, reduces duplication in the generated surface, and I think there are enough special behaviors and requirements that autogeneration would be tricky.